fpo-web/package.json

```diff
@@ -13,6 +13,7 @@
   "private": true,
   "dependencies": {
     "@ambroise-rabier/ngx-matomo": "1.0.10",
+    "@angular-devkit/core": "^10.0.1",
```

Comment:
You're including devkit from angular 10 instead of angular 8, but also it's not needed.